index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My porfolio</title>
+</head>
+<body>
+    <header>
+        <div class="nav-container">
+            <nav>
+                <img class="logo" src="images/John Doe (1).png" alt="">
+                <img class="nav-icon" src="images/Union.png" alt="">
+            </nav>
+        </div>
+        <h1>
+            Hey There. I’m Nana
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <div class="social-icons">
+            <ul>
+                <li><a href="#"><img src="images/Vector4.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Group.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector2.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector1.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector.png" alt=""></a></li>
+            </ul>
+        </div>
```

Comment:
- I think it will be better if you wrap this entire section in a `<section class=" introduction">` tag, as this section contains an introduction about you, your bio, what you do, and social media handles. 

- I believe this part needs to have a background image as well. You can simply add a background image on your CSS file after wrapping up your entire introduction in a section tag. 🍷 

| Figma Template Design  |Your Design |
| :---:   | :---: | 
|![desktopbg](https://user-images.githubusercontent.com/99246359/198818755-7d98c505-9a16-4964-8ec3-571d5525ebba.PNG)  |![hehas](https://user-images.githubusercontent.com/99246359/198818773-8862d73c-fa92-4f25-9588-a069bdb09224.PNG)  |  


- I think you have the image saved in your images folder as `illustration-header-mobile.png` 🌸 

So you can do this 👇🏽 

```
.introduction {
  other styles: here;
  background: url("images/illustration-header-mobile.png");
}

```